### DISCUSSION TITLE
Ask problems about branching

### DISCUSSION THREAD

**Discussion by cyLi-Tiger** [apache/airflow#22376] [General]
### What do you see as an issue?

If I have a dag looks like attached: a branching with two branches, I wonder which branch will be executed first? It seems that they won't be executed at the same time. And how can I control which branch to go first?
<img width="276" alt="image" src="https://user-images.githubusercontent.com/37681002/159121949-0631bf85-05ff-48f1-a4a6-8e02821657a3.png">

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Please DO NOT open issue when you want to ask questions. You were very clearly guided you should open discussion instead.

**Reply by potiuk** [reply]
You have no control. They might be executed either in parallell or not - depending on how busy your system is and which executor you use. If you want to have sequential execution make dependency between the tasks you want to run sequentially.

**Reply by cyLi-Tiger** [reply]
thanks a lot, I will read the guidance more carefully next time.
